zproject/prod_settings_template.py

```diff
@@ -152,6 +152,23 @@
     # "zproject.backends.GenericOpenIdConnectBackend",  # Generic OIDC integration, setup below
 )
 
+## JWT-based users login and API keys fetching.
+##
+## If you want to login users or fetch API keys via JWT in Zulip,
+## you need to set the secret key and algorithm to use to validate
+## JWT tokens received.
```

Comment:
We should probably rework this to have a bit more documentation in `docs/authentication-methods.md`, with this just being a stub linking to there?

Reply:
Also, this should definitely not be the first block in this file; let's move it to be the last of the authentication blocks in this file.

Reply:
I wrote up a brief doc in `authentication-methods.md` and pushed.